libs/cli/langgraph_cli/config.py

```diff
@@ -17,27 +18,46 @@ class Config(TypedDict):
 
 
 def validate_config(config: Config) -> Config:
-    config = {
-        "python_version": config.get("python_version", "3.11"),
-        "pip_config_file": config.get("pip_config_file"),
-        "dockerfile_lines": config.get("dockerfile_lines", []),
-        "dependencies": config.get("dependencies", []),
-        "graphs": config.get("graphs", {}),
-        "env": config.get("env", {}),
-    }
-    if config["python_version"] not in (
-        "3.11",
-        "3.12",
-    ):
-        raise click.UsageError(
-            f"Unsupported Python version: {config['python_version']}. "
-            "Supported versions are 3.11 and 3.12."
-        )
-    if not config["dependencies"]:
-        raise click.UsageError(
-            "No dependencies found in config. "
-            "Add at least one dependency to 'dependencies' list."
-        )
+    config = (
+        {
+            "node_version": config.get("node_version"),
+            "dockerfile_lines": config.get("dockerfile_lines", []),
+            "graphs": config.get("graphs", {}),
+            "env": config.get("env", {}),
+        }
+        if config.get("node_version")
+        else {
+            "python_version": config.get("python_version", "3.11"),
+            "pip_config_file": config.get("pip_config_file"),
+            "dockerfile_lines": config.get("dockerfile_lines", []),
+            "dependencies": config.get("dependencies", []),
+            "graphs": config.get("graphs", {}),
+            "env": config.get("env", {}),
+        }
+    )
+
+    if config.get("node_version"):
+        if config["node_version"] not in ("20"):
```

Comment:
`("20")` needs to be `("20",)` -- i am surprised this is not raising an error right now